source/install/prepare-mattermost-database.rst

```diff
@@ -204,3 +207,25 @@ When the PostgreSQL database is installed, and the initial setup is complete, yo
 .. important::
 
   If you are upgrading a major version of Postgres, it is essential that ``ANALYZE VERBOSE`` is run on the database post upgrade. This is necessary to re-populate the ``pg_statistics`` table used to generate optimal query plans. The database performance might suffer if this step is not done.
+
+
+Minimum supported version policy
+---------------------------------
+
+To make planning easier and ensure your Mattermost deployment remains fast and secure, we are introducing a policy for updating the minimum supported version of Postgres. The oldest supported Postgres version Mattermost supports will match the oldest version supported by the Postgres community. This ensures you benefit from the latest features and security updates.
+
+This policy change takes effect from Mattermost v10.6, where the minimum Postgres version required will be Postgres 13. This aligns with the Postgres community's support policy, which provides 5 years of support for each major version.
+
+.. note::
+
+  Mattermost v10.6 is not an :ref:`Extended Support Release (ESR) <about/release-policy:extended support releases>`. Going forward, this database version support policy will only apply to ESR releases. When a Postgres version reaches its end of life (EOL), Mattermost will require a newer version starting with the next scheduled ESR release.
```

Comment:
Wondering if having a bullet list or table with a map of the future support plan could be helpful (to us and customers alike). For example, if v10.11 is the next v10 ESR, we plan to bump Postgres min version to v14.

Reply:
I like this suggestion.

Reply:
@amyblais - Which server versions between v10.5 and v11 are ESRs? I'll incorporate these details into this docs PR.

Reply:
Confirmed with Eric that v10.5 and v.11 be our next ESRs ahead of v11. Given that this policy is being introduced with v10.6, the next ESR release would be v10.11, and that's where we'd bump the minimum version of PostgreSQL to v14. I'll incorporate this update into this docs PR.